File: parent335/pinyougou-web-shop/src/main/webapp/js/service/seckillgoodsService.js
//服务层
app.service('seckillgoodsService',function($http){
	    	

	//分页 
	this.findPage=function(page,rows){
		return $http.get('../seckillgoods/findPage.do?page='+page+'&rows='+rows);
	}

	//查询实体
	this.findOne=function(id){
		return $http.get('../seckillgoods/findOne.do?id='+id);
	}
	//增加
	this.add=function(entity){
		return  $http.post('../seckillgoods/add.do',entity );
	}

	//搜索
	this.search=function(page,rows,searchEntity){
		return $http.post('../seckillgoods/search.do?page='+page+"&rows="+rows, searchEntity);
	}    	
});

File: parent335/pinyougou-interface/src/main/java/cn/itcast/core/service/seckill/SeckillGoodsService.java
package cn.itcast.core.service.seckill;

import cn.itcast.core.entity.PageResult;
import cn.itcast.core.pojo.good.Goods;
import cn.itcast.core.pojo.seckill.SeckillGoods;

import java.util.Map;

public interface SeckillGoodsService {
    /**
     * 查询所有商品
     * @param page
     * @param rows
     * @param seckillGoods
     * @return
     */
    PageResult search(Integer page, Integer rows, SeckillGoods seckillGoods);

    /**
     * 商品申请秒杀
     * @param map
     */
    void add(Map map, String sellerId);

    /**
     *
     * @param seckillGoods
     */
    void addNewSeckill(SeckillGoods seckillGoods);
}
